### ISSUE TITLE
Use modern HTML5 and strict JS

### ISSUE THREAD

**Issue by Rangi42** [gbdev/rgbds-live#50]
This would mean adding `<!DOCTYPE html>` in the HTML and `"use strict";` in the JS, and following modern best practices (e.g. use `const` and `let` instead of `var`).

**Comment by SelvinPL** [comment]
Are we choosing any framework or just go wild with plain javascript?

**Comment by ISSOtm** [comment]
I don't think we need a framework beyond ACE, if you consider it one?

**Comment by Rangi42** [comment]
I do not plan to add React or anything like that.

**Comment by avivace** [comment]
Yeah, it's pretty clear that mantaining the codebase in plain js is going to be difficult / time consuming so I'd suggest picking either Vue and React and start adapting the code. Do we have any preference between the two?

**Comment by SelvinPL** [comment]
@Rangi42 sorry, got too much work... Since I don't learn either I'm fine with React... Do you know good component library (to not reinvent menus, tabs etc.)
